### PR TITLE
Documenting use of socat to access serial console

### PR DESCRIPTION
**Pull request by rd4398**
I have added the command to access serial console using socat.
Please let me know if anything else needs to be added!

### CODE REVIEW

**Comment by rd4398** [comment]
Yeah, will add!

**Comment by rd4398** [comment]
I have updated the documentation as per your feedback!
Please let me know if anything else needs to be added / changed.
Also, let me know when I should test the serial console for all nodes again in moc-esi!

**Comment by tzumainn** [comment]
It probably makes sense to put the new admin-stuff at the front of the section, since it's a pre-requisite.

File: docs/source/usage/cli.rst [hunk 1]
```diff
@@ -243,9 +243,18 @@ serial console information for a node from the Bare Metal service as follows:
...
+If ``console_enabled`` is ``true``, we can access the serial console using following command:
+
+``socat - tcp:<host>:<port>``
+
 If ``console_enabled`` is ``false`` or ``console_info`` is ``None`` then
 the serial console is disabled. Note, there can only be one ipmi connection to the node, meaning only one user may access the console at a time.
 
+Note that for serial console to function:
+
+* The ``ipmi_terminal_port`` must be set by the admin. This port must be unique per node/controller.
```
Comment (tzumainn) [inline docs/source/usage/cli.rst:255]: After further experimentation, it's safest to just say that the port must be unique.

Comment (rd4398) [reply docs/source/usage/cli.rst:255]: Ah, okay will make changes!

**Comment by rd4398** [comment]
Something like the latest commit?

File: docs/source/usage/cli.rst [hunk 2]
```diff
@@ -216,6 +216,11 @@ In order to access a node using a serial console, the admin or owner must config
 Instructions for this can be found under `Configuring Web or Serial Console`_ in Ironic's documentation.
 Once the node is properly configured, the console can be enabled and disabled as needed.
 
+Following is the pre-requisite for functioning of serial console:
+
+* The ``ipmi_terminal_port`` port must be unique per node/controller.
```
Comment (tzumainn) [inline docs/source/usage/cli.rst:221]: It should be noted that only the admin or node owner can set this value. Also, let's just say that the port must be unique, period.

**Comment by rd4398** [comment]
Okay!
I changed it...

**Comment by tzumainn** [comment]
Thanks!

**Pull request merged**